package.json

```diff
@@ -1,8 +1,10 @@
 {
   "license": "MIT",
   "devDependencies": {
-    "parcel-bundler": "^1.12.5",
-    "purescript": "^0.14.3",
+    "esbuild": "^0.14.49",
+    "parcel": "^2.6.0",
+    "process": "^0.11.10",
+    "purescript": "^0.15.3",
     "spago": "^0.20.3"
   },
   "dependencies": {
```

Comment:
https://github.com/JordanMartinez/purescript-cookbook/pull/292#discussion_r922500876

Reply:
Btw. I can offer to prepare a pull request for react v18. Since the changes are already merged (https://github.com/lumihq/purescript-react-basic-dom/pull/44) the repository can be prepared by overriding the dependency, updating the react examples and be merged as soon as the package-set is ready.